src/components/ProductForm/ProductForm.spec.js

```diff
@@ -14,43 +14,49 @@ describe('ProductForm', () => {
     const heading = screen.getByRole('heading')
     expect(heading).toBeInTheDocument()
 
-    const nameInput = screen.getByLabelText('Product Name')
+    const nameInput = screen.getByPlaceholderText('Enter product name..')
```

Comment:
Please keep the `getByLabelText` query. Even though it is hidden for users, it should still be rendered in our DOM.

Reply:
Maybe because your `label` is not connected to the input?